buildconfig/Jenkins/Conda/conda-build-recipes

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash -ex
+
+# This script expects to be in a POSIX environment, the script clones the conda-recipes
+# repo and then runs conda builds based on passed args. By default it only clones the
+# source, you need to pass a flag to actually build something useful. If building a
+# package dependent on a different one it will build and produce that package too.
+#
+# Script usage:
+# conda-build-recipes WORKSPACE [options]
+#
+# Example command to build all of the packages:
+# conda-build-recipes $WORKSPACE --build-mantid --build-qt --build-workbench
+#
+# Expected args:
+#   1. WORKSPACE: path to the workspace/source code that this should run inside
+#                 (mantid repo). Windows Caveat: Only use / for this arguement do
+#                 not use \\ or \ in the path.
+#
+# Possible flags:
+#   --build-mantid: build the mantid package using conda-build
+#   --build-qt: build the qt package using conda-build
+#   --build-workbench: build the workbench package using conda-build
+#
+# Possible parameters:
+#   --clobber-yml: clobber the meta yaml files of each package build with conda-build, provide an absolute path to the uteyml file
+#
+
+# Setup expected variables
+WORKSPACE=$1
+shift
+if [[ $OSTYPE == "msys"* ]]; then
+    WORKSPACE=${WORKSPACE////\\}
```

Comment:
Could we comment what this line is doing? It's *magic* that tends to be lost to time else